lib/package-detail-view.js

```diff
@@ -372,7 +372,16 @@ export default class PackageDetailView {
       readme = fs.readFileSync(this.readmePath, {encoding: 'utf8'})
     }
 
-    const readmeView = new PackageReadmeView(readme)
+    let readmeSrcBase
+
+    if (this.pack.path) {
+      readmeSrcBase = this.pack.path
+    } else {
+      let repoUrl = this.packageManager.getRepositoryUrl(this.pack)
```

Comment:
Edge case, but this might return undefined if the repository no longer exists or the package has been unpublished.  You might want to make sure that this works in those cases.

Reply:
Just to clarify, is it `getRepositoryUrl()` which could return undefined?

Reply:
In latest commit I've added some checks to make try and catch this